pkg/models/load_balancer.go

```diff
@@ -539,24 +716,26 @@ type CreateLBVirtualServers struct {
 }
 
 type CreateLBVirtualServersReq struct {
-	ID                  int                 `json:"id" tf:"id,computed"`
-	Description         string              `json:"description"`
-	VipName             string              `json:"vipName" tf:"vip_name"`
-	VipAddress          string              `json:"vipAddress" tf:"vip_address"`
-	VipProtocol         string              `json:"vipProtocol" tf:"vip_protocol"`
-	VipPort             string              `json:"vipPort" tf:"vip_port"`
-	Pool                int                 `json:"pool"`
-	SSLServerCert       int                 `json:"sslServerCert" tf:"ssl_server_cert"`
-	SSLCert             int                 `json:"sslCert" tf:"ssl_cert"`
-	VirtualServerConfig VirtualServerConfig `json:"config"`
+	ID                  int                  `json:"id" tf:"id,computed"`
+	LbID                int                  `json:"-" tf:"lb_id"`
+	Description         string               `json:"description" tf:"description"`
+	VipName             string               `json:"vipName" tf:"name"`
+	VipAddress          string               `json:"vipAddress" tf:"vip_address"`
+	VipProtocol         string               `json:"vipProtocol" tf:"type"`
+	VipPort             string               `json:"vipPort" tf:"vip_port"`
+	Pool                int                  `json:"pool" tf:"pool"`
+	VipHostName         string               `json:"vipHostName" tf:"vip_host_name"`
+	SSLServerCert       int                  `json:"sslServerCert" tf:"ssl_server_cert"`
+	SSLCert             int                  `json:"sslCert" tf:"ssl_client_cert"`
+	VirtualServerConfig *VirtualServerConfig `json:"config" tf:"config,sub"`
```

Comment:
I would assume that you will take care of servers and pools and profiles related changes in the subsequent review. Please remove duplicate JSON keyword related to them in subsequent review. Please take care of it later.

Reply:
yes